### PR TITLE
[SPARK-8549][SparkR] Fix the line length of SparkR

### PR DESCRIPTION
**Pull request by yu-iskw**
[[SPARK-8549] Fix the line length of SparkR - ASF JIRA](https://issues.apache.org/jira/browse/SPARK-8549)

### CODE REVIEW

**Comment by yu-iskw** [comment]
@shivaram Could you review this PR when you have time? Thanks!

File: R/pkg/R/utils.R [hunk 1]
```diff
@@ -334,18 +334,21 @@ getStorageLevel <- function(newLevel = c("DISK_ONLY",
                                          "MEMORY_ONLY_SER_2",
                                          "OFF_HEAP")) {
   match.arg(newLevel)
+  strageLevelClass <- "org.apache.spark.storage.StorageLevel"
```
Comment (shivaram) [inline R/pkg/R/utils.R]: can we call this `storageLevelClass` ?

Comment (yu-iskw) [inline R/pkg/R/utils.R]: No. It is a just string variable.

Comment (shivaram) [inline R/pkg/R/utils.R]: Yeah but the variable name spelling is strange right now as `strage` -- could you make the variable name `storage` instead ?

Comment (yu-iskw) [inline R/pkg/R/utils.R]: Sorry about that. I got it.

**Comment by shivaram** [comment]
Thanks @yu-iskw -- Just for my information can you paste a link to output of `dev/lint-r` after this change ?

**Comment by yu-iskw** [comment]
@shivaram , Sure. That is the result of `./dev/lint-r` at the revision `d9838196ff48faeac19756852a7f695129c08047`. Could you check it?
https://gist.github.com/yu-iskw/d3f414f2c18b2abc5766

**Comment by shivaram** [comment]
Thanks @yu-iskw. BTW We should investigate how to get rid of the false warnings `no visible global function definition` as those functions are package private.

But this change LGTM. Merging this

**Comment by yu-iskw** [comment]
@shivaram Thank you for merging it!
I agree with that. I will check the lintr code and ask Jim it.

**Comment by rxin** [comment]
@yu-iskw you might want to update your github email to include the email address you used for your commits. Otherwise it doesn't show up in the github metadata.

**Comment by yu-iskw** [comment]
@rxin Thank you for letting me know. I have updated my github email in my account profile.

**Pull request closed without merging**